gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/ManipulateZonesAndRecordSets.java

```diff
@@ -23,6 +23,7 @@
 package com.google.gcloud.examples.dns.snippets;
 
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
> Here it is needed for iteration.

You mean for accessing `Status`? As @aozarov noted `Status` is a member of `ChangeRequestInfo`. Also if we add `isDone()` iteration becomes:
```java
ChangeRequest request = zone.applyChangeRequest(changeRequest);
while (!request.isDone()) {
  Thread.sleep(500L);
}
```